x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/column_types.ts

```diff
@@ -19,17 +19,18 @@ export interface BaseIndexPatternColumn extends Operation {
   timeShift?: string;
 }
 
-export interface FormatParams {
+export interface LensFormatParams {
```

Comment:
Do I understand correctly that this interface defines configuration to format values for display to the user?

If so, we could consider calling it `DisplayFormatConfig` or something. I'm not sure what we gain by prepending `Lens` since we're already in the `lens` plugin.

Reply:
Maybe `ValueFormatConfig`?

Reply:
I like it!